### ISSUE TITLE
Finetune stage

### ISSUE THREAD

**Issue by seongwn** [jxhuang0508/CaCo#8]
Hi! I'm embarrassed to leave a lot of issues :)

I leave an issue because I have a question about your training procedure. 
After CaCo warmup stage, you fine-tune the model with ProDA, CRS, or FDA methodology. 
I wonder if the CaCo objective function is not used in the fine-tuning process.

Even when I look at the command lines for CaCo + ProDA, I cannot find where CaCo is used. (From warm-up stage to Stage 3)

Thanks a lot.
